pysd/py_backend/functions.py

```diff
@@ -504,6 +504,10 @@ def __init__(self, py_model_file):
         self.time.stage = 'Load'
         self.initialize()
 
+    def getPysdCompilerVersion(self):
```

Comment:
This may want to live in the Macro class, in case we want to know if  a macro was translated with a different version than the main model. An edge case, so not very important. What do you think?

Reply:
Also, lets use `get_pysd_compiler_version` to be similar to the other methods in the object...

Reply:
Changed